### PR TITLE
Remove placeholder brackets from twitter payload

### LINKED ISSUE #576: [Bug] Remove placeholder brackets  from Twitter payload
# Bug Report

### System

- OS: macos mojave 
- OS Version: 10.14.6

### Browser

- Browser: Chrome
- Browser Version: 78.0.3904.108 (Official Build) (64-bit)

### Maskbook

- Maskbook Version:
- Installation: Self-Complied
- Build Commit: 15482abc929765223adb108d7f7f17bc056f6b51

### Expect Behavior

image

### PR DESCRIPTION
**Pull request by guanbinrui**
Close #576

### CODE REVIEW

File: src/social-network-provider/twitter.com/ui/index.ts [hunk 1]
```diff
@@ -21,12 +21,12 @@ export const instanceOfTwitterUI = defineSocialNetworkUI({
...
         en: {
             additional_post_box__encrypted_post_pre: {
-                message: '#Maskbook ([I:b])\nDecrypt this tweet with maskbook.com @ProjectMaskbook.\n—§— /* {$1} */',
+                message: '#Maskbook ([I:b])\nDecrypt this tweet with maskbook.com @ProjectMaskbook.\n—§— $1',
             },
         },
         zh: {
             additional_post_box__encrypted_post_pre: {
-                message: '#Maskbook ([I:b])\n使用 maskbook.com @ProjectMaskbook 解密这条推文。\n—§— /* {$1} */',
+                message: '#Maskbook ([I:b])\n使用 maskbook.com @ProjectMaskbook 解密这条推文。\n—§— $1',
```
Comment (SunriseFox) [inline src/social-network-provider/twitter.com/ui/index.ts]: I think `/* */` should stay... It indicates it is a comment between them...

**Comment by Tedko** [comment]
What do you mean ‘comment’?

File: src/social-network-provider/twitter.com/ui/index.ts (hunk 1 above)
Comment (guanbinrui) [reply src/social-network-provider/twitter.com/ui/index.ts]: > I think `/* */` should stay... It indicates it is a comment between them...

Here is the combinations, and I think you are right let's add `/* */` stuff back?

image

**Comment by guanbinrui** [comment]
> What do you mean ‘comment’?

After reread #373. I think the author wanna use comment notes expressing those wrapped stuff is not important in the tweet.

**Pull request merged**